generate_csv.py

```diff
@@ -2,126 +2,46 @@
 from tqdm import tqdm
 import os
 import shutil
-from openfisca_uk.tools.simulation import model
 import numpy as np
-
-BENEFITS = {
-    1: "DLA_SC",
-    2: "DLA_M",
-    3: "child_benefit",
-    4: "pension_credit",
-    5: "state_pension",
-    6: "BSP",
-    8: "AFCS",
-    9: "war_pension",
-    10: "SDA",
-    12: "AA",
-    13: "carers_allowance",
-    14: "JSA",
-    14.1: "JSA_contrib",
-    14.2: "JSA_income",
-    15: "IIDB",
-    17: "incapacity_benefit",
-    19: "income_support",
-    21: "maternity_allowance",
-    37: "guardians_allowance",
-    36: "GTA",
-    30: "other_benefit",
-    90: "working_tax_credit",
-    91: "child_tax_credit",
-    92: "WTC_lump_sum",
-    93: "CTC_lump_sum",
-    94: "housing_benefit",
-    69: "SFL_IS",
-    70: "SFL_JSA",
-    111: "SFL_UC",
-    62: "winter_fuel_allowance",
-    65: "DWP_IS",
-    66: "DWP_JSA",
-    110: "DWP_UC",
-    24: "FG",
-    22: "MG",
-    60: "widows_payment",
-    98: "DWP_loan",
-    99: "LA_loan",
-    95: "universal_credit",
-    96: "PIP_DL",
-    97: "PIP_M"
-}
-
-BENEFITS = {code: benefit + "_reported" for code, benefit in BENEFITS.items()}
-
-AGES = [2, 7, 13, 18, 22, 27, 32, 37, 42, 47, 52, 57, 62, 67, 72, 77, 82, 90]
-
-JSA_ESA_TYPES = {
-    0: "income",
-    1: "income",
-    2: "income",
-    3: "contrib",
-    4: "contrib",
-    5: "income",
-    6: "income"
-}
+from frs_params import *
 
 PERSON_FIELDNAMES = [
     "person_id",
     "benunit_id",
     "household_id",
     "role",
-    "is_male",
-    "is_adult",
-    "is_child",
-    "age",
-    "is_head",
-    "is_householder",
-    "employee_earnings",
-    "deductions",
-    "self_employed_earnings",
+    "adult_weight",
+    "earnings",
+    "profit",
+    "childcare",
     "pension_income",
-    "total_benefits",
-    "interest",
-    "assets",
-    "maintenance_expense",
+    "age",
+    "care_hours",
+    "hours",
+    "savings_interest",
     "misc_income",
-    "JSA_contrib_eligible",
-    "disabled",
-    "adult_weight",
-    "hours_worked",
-    "actual_net_income",
-    "student_loan_repayments",
-    "net_income_adjustment",
-    "eligible_childcare_cost",
-    "SSP",
-    "is_adult_1",
-    "total_disability_benefits"
-] + list(BENEFITS.values())
+    "total_benefits",
+    "is_household_head",
+    "is_benunit_head",
+    "FRS_net_income"
+] + [benefit for benefit in BENEFITS.values() if benefit not in BENUNIT_LEVEL_BENEFITS and benefit in REPORTED] + [benefit + "_reported" for benefit in BENEFITS.values() if benefit not in BENUNIT_LEVEL_BENEFITS and benefit in SIMULATED and benefit]
 
 BENUNIT_FIELDNAMES = [
     "benunit_id",
-    "benunit_weight",
-    "external_child_maintenance"
-]
+    "benunit_weight"
+] + [benefit for benefit in BENEFITS.values() if benefit in BENUNIT_LEVEL_BENEFITS and benefit in REPORTED] + [benefit + "_reported" for benefit in BENEFITS.values() if benefit in BENUNIT_LEVEL_BENEFITS and benefit in SIMULATED]
```

Comment:
any way to shorten this line?

Reply:
Just used the formatter on it, should add that as a GitHub action for this repo